sdk/core/Azure.Core/Azure.Core.sln

```diff
@@ -57,6 +57,12 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Net.ClientModel", ".
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Net.ClientModel.Tests", "..\System.Net.ClientModel\tests\System.Net.ClientModel.Tests.csproj", "{180BE182-BB02-4505-9C8E-EA077D6917B0}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Net.ClientModel.Tests.Internal", "..\System.Net.ClientModel\tests\internal\System.Net.ClientModel.Tests.Internal.csproj", "{15D02938-4A22-4C87-BF37-464E932671C2}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Net.ClientModel.Tests.Clients", "..\System.Net.ClientModel\tests\clients\System.Net.ClientModel.Tests.Clients.csproj", "{F7C567CE-4A30-4674-8874-3D1D9058B7F0}"
+EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.Net.ClientModel.Tests.Perf.Internal", "..\System.Net.ClientModel\tests\perf.internal\System.Net.ClientModel.Tests.Perf.Internal.csproj", "{0935301F-D99D-499D-A0B2-CF48287597D7}"
```

Comment:
Will System.Net.ClientModel.Tests.Clients not work for that purpose?  I had thought that was what we agreed on for model code, since models are part of clients.

Reply:
Per offline discussion, it sounds like this is ok, but I will rename it to "Client" not "Clients" so it doesn't sound like it always has to be fully-implemented clients.  If we need common test components unrelated to clients in the future, we can reopen the conversation about a "Tests.Common" project.